test/tests/inittest.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+# This have to be first test for cockpit
+#  install build dependencies
+#  download new version of cockpit from git 
+#  compile it
+# it is basic test compilation must PASS
+
+set -x
+
+BUILD="build1"
+PACKAGE="cockpit"
+SOURCE="https://github.com/$PACKAGE-project/$PACKAGE.git"
+
+#
+yum-builddep -y $PACKAGE
+yum-builddep -y $PACKAGE
+
+ /bin/rm -fr $PACKAGE
+ git clone $SOURCE && \
```

Comment:
yes,
Now it is more less only demonstrating how to it  works with actual cockpit version.
Then this part will somehow solved: 
 * It will be connected with pull request not with master/origin
 * Now we can solve it  by copying itself to machine to use same version as test is (copy local version of cockpit -> guest).
 * Final solution, from my point of view is, that there will be builded RPMs for thhat purposes:
pull request -> http://openbuildservice.org/ -> install rpm on host + guests -> test it
compilation test is only special testcase, probably then this compilation test will be done via installing srpm and compiling.